modules/asg_node_group/variables.tf

```diff
@@ -178,3 +178,9 @@ variable "bottlerocket_admin_container_source" {
   default     = ""
   description = "URI of a custom admin container image"
 }
+
+variable "imdsv2_enabled" {
```

Comment:
I think name like "required" or "enforced" would be more descriptive

Reply:
changed to imdsv2_required

Reply:
changed to imdsv2_required